packages/pulp/pulp.spec

```diff
@@ -381,9 +381,7 @@ Requires: pulp-selinux
 Requires: python-%{name}-common = %{pulp_version}
 Requires: python-%{name}-repoauth = %{pulp_version}
 Requires: python-blinker
-Requires: python2-celery >= 4.0.0
-Requires: python2-celery < 4.1
-Conflicts: python2-celery >= 4.1
+Requires: python-celery >= 4.0.0
```

Comment:
Should this be `10:4.0.0` now?

Reply:
eh, theoretically, yes.  But since the pulp repo will be enabled for this to be installed, and the pulp repo contains the high epoch setting, it should be ok in this situation.